contracts/margin/interfaces/LoanOfferingVerifier.sol

```diff
@@ -31,6 +31,35 @@ pragma experimental "v0.5.0";
  *       to these functions
  */
 contract LoanOfferingVerifier {
+
+    // ============ Structs ============
+
+    struct LoanOffering {
```

Comment:
This is different than the `LoanOffering` in `MarginCommon` which is confusing. It's nice this doesn't use the dumb rates thing, but we should make them both the same